fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisManager.java

```diff
@@ -157,6 +158,11 @@ public AnalysisManager() {
             this.taskExecutor = new AnalysisTaskExecutor(Config.statistics_simultaneously_running_task_num,
                     Integer.MAX_VALUE);
             this.statisticsCache = new StatisticsCache();
+            this.dropStatsExecutors = ThreadPoolManager.newDaemonThreadPool(
+                1, 1, 0,
+                TimeUnit.DAYS, new LinkedBlockingQueue<>(10),
```

Comment:
the capacity 10 is enough? what will happen after queue is full?

Reply:
I think 10 is enough. Truncate table operation is usually not very frequently and not very heavy unless it's a table with many partitions and columns. When the queue is full, new request will simply aborted. It's not big problem. The only impact is the cached stats will not be invalidated after truncate operation, but after truncating, the table is not heathy anymore, it will be auto analyzed next time and the cache will be updated after that.